server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/SetMessages.feature

```diff
@@ -100,4 +100,26 @@ Feature: SetMessages method on shared folders
     Given "bob@domain.tld" shares his mailbox "shared" with "alice@domain.tld" with "latires" rights
     And "bob@domain.tld" sets flags "$Flagged" on message "mBob"
     When "alice@domain.tld" sets flags "" on message "mBob"
-    Then message "mBob" is not updated
\ No newline at end of file
+    Then message "mBob" is not updated
+
+# Updating draft
+
+  Scenario: A user can update the flags on a draft
+    Given "bob@domain.tld" has a mailbox "Drafts"
+    And "bob@domain.tld" creates a draft message "mDraft"
+    When "bob@domain.tld" sets flags "$Draft,$Seen" on message "mDraft"
+    Then "bob@domain.tld" should see message "mDraft" with keywords $Draft,$Seen
+
+  Scenario: A user can not remove a draft flag on a draft messages
+    Given "bob@domain.tld" has a mailbox "Drafts"
+    And "bob@domain.tld" creates a draft message "mDraft"
+    When "bob@domain.tld" sets flags "$Seen" on message "mDraft"
+    Then message "mDraft" is not updated
+    And "bob@domain.tld" should see message "mDraft" with keywords $Draft
+
+  Scenario: A user can update the flags on a draft
+    Given "bob@domain.tld" has a mailbox "Drafts"
+    And "bob@domain.tld" creates a draft message "mDraft"
+    When "bob@domain.tld" destroys message "mDraft"
```

Comment:
you mean remove flag?

Reply:
I mean "destroy". The scenario line is wrong.